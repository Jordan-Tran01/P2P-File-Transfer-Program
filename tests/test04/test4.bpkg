ident:5105d1a7ffbde836fe5aaa6704ecc751857561c6073
filename:test3.data
size:131072
nhashes:3
hashes:
	e124c67d39b9fad2d3c06b0d6a3bb1861ebfa66fd0dacfea7caa53682c8c6498
	7ca3ff8cd899a382896b0f2590a0aa8cf1137c4a2a08a728d864ec75576ec0ed
	9ab3c810224bdb8cb14caed9031a25e567fa9fc1055d84f31b91ee7d1fb2c12c
nchunks:4
chunks:
	2c87207bc909188bb45904db002f7eb6da05e5d3031fbf0834a230d1d3691c61,0,32768
	904fefa128f596dfb144e35faa7b7faefae93cd8d83847fb249d144f97e7ceb9,32768,32768
	677b489f36eabd83be1a816da3e5608407a817a7f7d6c2cccb95c58e3f8af6e3,65536,32768
	eb08794e141762caebe269e7c23418befd57a73477fa658a7cf845fb21719e70,98304,32768
