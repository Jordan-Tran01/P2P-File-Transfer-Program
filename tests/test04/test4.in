CONNECT 127.0.0.1:9856
ADDPACKAGE pkg4.bpkg
FETCH 127.0.0.1:9856 5105d1a7ffbde836fe5aaa6704ecc751857561c6073 2c87207bc909188bb45904db002f7eb6da05e5d3031fbf0834a230d1d3691c61