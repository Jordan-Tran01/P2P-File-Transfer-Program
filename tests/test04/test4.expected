Connection established with peer
Peer failed to send requested data
